ddtrace/ext/messaging.go

```diff
@@ -0,0 +1,24 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023 Datadog, Inc.
+
+package ext
+
+const (
+	// MessagingSystem identifies which messaging system created this span (kafka, rabbitmq, amazonsqs, googlepubsub...)
+	MessagingSystem = "messaging.system"
+)
+
+const (
+	MessagingSystemGCPPubsub = "googlepubsub"
+	MessagingSystemKafka     = "kafka"
+)
+
+// Kafka tags.
+const (
+	// MessagingKafkaPartition defines the Kafka partition the trace is associated with.
+	MessagingKafkaPartition = "messaging.kafka.partition"
+	// KafkaBootstrapServers holds a comma separated list of bootstrap servers as defined in producer or consumer config.
+	KafkaBootstrapServers = "messaging.kafka.bootstrap.servers"
+)
```

Comment:
I'm trying to follow similar approach as in other files (see `db.go`), where we are also trying to separate in the different sections that we have in our internal docs.

The idea of the existing 3 sections in this file is the following:

- 1st section: messaging "core" tags - currently just `messaging.system`
- 2nd section: available values for the `messaging.system` tag.
- 3rd section: kafka specific ones.

 Since the file is dedicated to `messaging`, I thought this would be a good idea to group constants. Please let me know if you prefer a different organization, or whatever you have in mind 🙂